app/index.js

```diff
@@ -1,4 +1,5 @@
 import { app, dialog, shell, ipcMain } from 'electron'
+import './prepare-app'
```

Comment:
Should we add this in the `index.js` or in the `entry.js`? Where does it make more sense?

Reply:
Just tested and both approaches work, I don't have a preference tbh. What do you think?